tox.ini

```diff
@@ -124,7 +126,8 @@ allowlist_externals = sh
 setenv =
   ; override CONTRIB_REPO_SHA via env variable when testing other branches/commits than main
   ; i.e: CONTRIB_REPO_SHA=dde62cebffe519c35875af6d06fae053b3be65ec tox -e <env to test>
-  CONTRIB_REPO_SHA={env:CONTRIB_REPO_SHA:main}
+  ; CONTRIB_REPO_SHA={env:CONTRIB_REPO_SHA:main}
+  CONTRIB_REPO_SHA=e5b193aec4acf6528f3e01d03220ee5e2ded50cf
```

Comment:
I'll remove this once https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2687 gets merged. It is necessary for the time being for the tests to pass.